src/api/python/_api.py

```
@@ -379,7 +389,12 @@ def get_backend_params(self, backend: str) -> dict[str, str]:
     @param initParams Dictionary of initialization parameters.
     """
 
-    def create_backend(self, backend: str, initParams: dict[str, str] = {}):
+    def create_backend(self, backend: str, initParams: Optional[dict[str, str]] = None):
```

Ruff (0.15.15)

[warning] Missing return type annotation for private function `create_backend`

Add return type annotation: `None`

(ANN202)